onnxmltools/convert/common/model_util.py

```diff
@@ -9,12 +9,44 @@
 from ...proto import onnx_proto
 from ...proto import helper
 
+
+
 # Includes basic ONNX data types only, advanced data types will be supported later.
 # This list is used to convert integers to floats.
 onnx_integer_types = [onnx_proto.TensorProto.UINT8, onnx_proto.TensorProto.INT8, onnx_proto.TensorProto.UINT16,
                       onnx_proto.TensorProto.INT16, onnx_proto.TensorProto.INT32, onnx_proto.TensorProto.INT64]
 
 
+def producer():
```

Comment:
Would it be better to rename producer to get_producer?

Reply:
I can go either way, but I see your point - I think the get term makes it sound that its retrieving rather than doing something. I will update.